src/games/strategy/engine/data/ChangeFactory.java

```diff
@@ -113,14 +105,6 @@ public static Change changeResourcesChange(final PlayerID player, final Resource
     return new ChangeResourceChange(player, resource, quantity);
   }
 
-  public static Change addResourceCollection(final PlayerID id, final ResourceCollection rCollection) {
```

Comment:
Will keep it around, but changed to private so we can know it's not used at least.